content/projects/gsoc/index.adoc

```diff
@@ -22,42 +22,43 @@ In exchange, numerous Jenkins community members volunteer as "mentors" for the s
 
 image:/images/gsoc/opengraph.png[Jenkins GSoC, role=center, float=center]
 
-== GSoC 2025
+== GSoC 2026
 
-// We are planning to participate in Google Summer of Code in 2025. +
+ We are planning to participate in Google Summer of Code in 2026. 
 
-We are participating in Google Summer of Code in 2025. +
+// We are participating in Google Summer of Code in 2026. +
 // See our link:https://docs.google.com/document/d/1FYOBo12qz24Vxq0TxWuv9ElHH_rHP51ouMsPms4tTmw/edit?usp=sharing[Jenkins GSoC Mentoring Org Application Form].
-Google has accepted us as a mentoring organization in Google Summer of Code 2025.
-See our link:https://summerofcode.withgoogle.com/programs/2025/organizations/jenkins-wp[Jenkins organization page on the GSoC website].
+// Google has accepted us as a mentoring organization in Google Summer of Code 2026.
+// See our link:https://summerofcode.withgoogle.com/programs/2025/organizations/jenkins-wp[Jenkins organization page on the GSoC website].
 
 // Uncomment when application is worked on and submitted (Feb 2025)
 //(link:./2025/application[Jenkins GSoC Organisation Application Form])
 
-The five selected projects for 2025 are:
+The draft projects for 2026 are:
 
-* link:/projects/gsoc/2025/projects/ai-powered-chatbot-for-quick-access-to-jenkins-resources[AI-Powered Chatbot for Quick Access to Jenkins Resources] with link:https://github.com/giovanni-vaccarino/[Giovanni Vaccarino] as the GSoC contributor.
 
-* link:/projects/gsoc/2025/projects/complete-alternative-jenkins-io-build-retooling[Complete build retooling of jenkins.io] with link:https://github.com/biru-codeastromer/[Birajit Saikia] as the GSoC contributor.
 
-* link:/projects/gsoc/2025/projects/domain-specific-LLM-based-on-jenkins-usage-using-ci-jenkins-io-data[Domain-specific LLM based on actual Jenkins usage using ci.jenkins.io data] with link:https://github.com/chiruu12/[Chirag Gupta] as the GSoC contributor.
+// * link:/projects/gsoc/2026/projects/complete-alternative-jenkins-io-build-retooling[Complete build retooling of jenkins.io].
 
-* link:/projects/gsoc/2025/projects/improving-tekton-client-plugin[Improving Tekton Client Plugin for Jenkins] with link:https://github.com/maeveho25/[Maeve Ho] as the GSoC contributor.
+// * link:/projects/gsoc/2026/projects/domain-specific-LLM-based-on-jenkins-usage-using-ci-jenkins-io-data[Domain-specific LLM based on actual Jenkins usage using ci.jenkins.io data].
 
-* link:/projects/gsoc/2025/projects/plugin-modernizer-improvements[Improving Plugin Modernizer] with link:https://github.com/CodexRaunak/[Raunak Madan] as the GSoC contributor.
+// * link:/projects/gsoc/2026/project-ideas/improving-tekton-client-plugin[Improving Tekton Client Plugin for Jenkins].
+* link:/projects/gsoc/2026/project-ideas/ai-powered-chatbot-for-quick-access-to-jenkins-resources[AI-Powered Chatbot for Quick Access to Jenkins Resources].
```

Comment:
```suggestion
// * link:/projects/gsoc/2026/project-ideas/ai-powered-chatbot-for-quick-access-to-jenkins-resources[AI-Powered Chatbot for Quick Access to Jenkins Resources].
```

Reply:
As discussed, plesase comment these all out for now so there will be no confusion as I will be working on them right away.